lib/mail/network/delivery_methods/sendmail.rb

```diff
@@ -61,7 +61,7 @@ def deliver!(mail)
 
     def self.call(path, arguments, destinations, encoded_message)
       popen "#{path} #{arguments} #{destinations}" do |io|
-        io.puts ::Mail::Utilities.to_lf(encoded_message)
+        io.puts ::Mail::Utilities.binary_unsafe_to_lf(encoded_message)
```

Comment:
Encoded message is 7bit ASCII, safe to convert newlines for `sendmail` (which expects LF newlines)